tests/layer_tests/requirements.txt

```diff
@@ -0,0 +1 @@
+networkx>=1.11
```

Comment:
Need to add necessary requirements for tf and onnx tests

Reply:
Why was the requirements file deleted?

Reply:
@just-sparta, all required packages including networkx are in model-optimizer requirements

Reply:
Should we add path to mo requirements in layer tests requirements file? Like as -r ../../model-optimizer/requirements.txt
@vurusovs

Reply:
It's expected that OpenVINO is configured right way (MO requirements are installed, IE PyAPI is configured). It sho uld be documented as prerequisites for tests in README